gen.go

```diff
@@ -18,8 +18,56 @@ func init() {
 	templates = template.Must(bundle.Parse(templates))
 }
 
-// Generate generates code according to the schema.
+func clean(gen []byte) ([]byte, error) {
+	// Remove blank lines added by text/template
+	bytes := newlines.ReplaceAll(gen, []byte(""))
+
+	// Format sources
+	clean, err := format.Source(bytes)
+	// TODO: If formatting errors, return the error instead of the broken,
+	// unformatted code
+	if err != nil {
+		return gen, err
+	}
+	return clean, nil
+}
+
 func (s *Schema) Generate() ([]byte, error) {
+	// Default to hyper schema for backwards compatibility
+	if s.Schema == nil {
```

Comment:
it's unfortunate we need this, but I thought a bit about it and there's no good way to avoid it...